tests/application_development/ram_context_for_isr/Kconfig

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2025 Muhmmad Waleed Badar
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+config TEST_IRQ_NUM
+	int "Test IRQ number"
+	default 42 if BOARD_QEMU_CORTEX_M3
+	default 14 if GIC
+	default 22 if SOC_SERIES_DA1469X
+	default 0
```

Comment:
Could you add `default 18 if SOC_SERIES_STM32C0X` [as you suggested](https://github.com/zephyrproject-rtos/zephyr/pull/94716#issuecomment-3219827076)?
(with `- STM32C0X series: 18 (available test IRQ)` below)